Jenkinsfile

```diff
@@ -12,67 +12,50 @@ pipeline {
       parallel {
         stage('Xenial') {
           agent {
-            kubernetes {
-              label 'mypod'
-              yaml """
-apiVersion: v1
-kind: Pod
-spec:
-  containers:
-    - name: 'buildenv-xenial'
-      image: 'geodynamics/virtualquake-buildenv-xenial:latest'
-      command:
-      - 'cat'
-      tty: true
-"""
+            docker {
+              image 'geodynamics/virtualquake-buildenv-xenial:latest'
+              alwaysPull true
             }
           }
+
           stages {
             stage('Prepare Environment (Xenial)') {
               steps {
-                container('buildenv-xenial') {
-                  sh '''sed -i '/[(79),(227),(239)]/s/${MPIEXEC}/${MPIEXEC} --allow-run-as-root/' examples/CMakeLists.txt'''
-                }
+                sh '''sed -i '/[(79),(227),(239)]/s/${MPIEXEC}/${MPIEXEC} --allow-run-as-root/' examples/CMakeLists.txt'''
```

Comment:
This is likely no longer necessary since #205 switches to MPICH, and this was an openmpi specific workaround.